.github/workflows/build.yml

```diff
@@ -11,6 +11,56 @@ on:
   pull_request:
 
 jobs:
+  stable:
+    name: "Test Stable"
```

Comment:
Nit: I don't like the "stable" name since it also tests nightly. Maybe we should name the job something with "msrv" or "features"?

Reply:
I'm thinking "Test MSRV and stable features"